CHANGELOG.md

```diff
@@ -666,7 +691,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 NOTE: These links should respect backports. See https://github.com/TheThingsNetwork/lorawan-stack/pull/1444/files#r333379706.
 -->
 
-[unreleased]: https://github.com/TheThingsNetwork/lorawan-stack/compare/v3.6.0...HEAD
+[unreleased]: https://github.com/TheThingsNetwork/lorawan-stack/compare/v3.7.0...HEAD
+[3.7.0]: https://github.com/TheThingsNetwork/lorawan-stack/compare/v3.6.2...v3.7.0
```

Comment:
Is GitHub smart enough to show a sensible diff here?

Reply:
No I wasn't sure of this myself. I think I'll just drop the comparison between backports and bumped minor (v3.6.2...v3.7.0).

Reply:
Would it make sense to compare minor versions here ex: v3.6.0...v3.7.0?

Reply:
I think in this case it should be `v3.6.0..v3.7.0` yes. I'm afraid there's not much else we can do